olive/passes/openvino/conversion.py

```diff
@@ -64,7 +64,7 @@ def _run_for_config(
 
         model_input_param = "input_model"
         input_model = model.model_path
-        if model.framework == Framework.TENSORFLOW and not model.is_file:
+        if model.framework == Framework.TENSORFLOW and model.model_type != ModelType.LocalFile:
```

Comment:
Nit: to be specified, it should be `model.model_type == ModelType.LocalFolder`. But in theory AML model is not supported for conversion now. I'll update this logic soon.

Reply:
will update